src/components/organisms/AuthenticationModal/LoginForm/LoginForm.tsx

```diff
@@ -120,6 +123,39 @@ const LoginForm: React.FunctionComponent<PropsType> = ({
             <span className="input-error">Password is required</span>
           )}
         </div>
+        <div className="input-group">
+          <input
+            name="date_of_birth"
+            className="input-block input-centered"
+            type="date"
+            ref={register}
+          />
+          {errors.date_of_birth && errors.date_of_birth.type === "required" && (
+            <span className="input-error">Date of birth is required</span>
+          )}
+        </div>
+        <div className="input-group">
+          <input
+            name="code"
+            className="input-block input-centered"
+            type="code"
+            placeholder="Ticket Code From Your Email"
+            ref={register({
+              required: true,
+            })}
+          />
+          {errors.code && (
+            <span className="input-error">
+              {errors.code.type === "required" ? (
+                <>
+                  Enter the ticket code from your email. The code is required.
+                </>
+              ) : (
+                errors.code.message
+              )}
+            </span>
+          )}
```

Comment:
This block seems as though it will show all errors, not just `required`, but the above 1 line check seems like it won't necessarily display all errors? Is the one liner correct?

Reply:
It'll display all the errors because it'll default to the message unless the error is required. This code just makes the required error more informative and passes others through.

Reply:
In my recent changes, I've also pushed a change that unifies both form fields to behave the same way at this, since their behaviour appears to be intended to be identical. The DRY-up exposed the similarity.